src/core/error_helpers.js

```diff
@@ -253,6 +406,8 @@ function helpForMisusedAtTopLevelCode(e, log) {
 
 // Exposing this primarily for unit testing.
 p5.prototype._helpForMisusedAtTopLevelCode = helpForMisusedAtTopLevelCode;
+p5.prototype._validateParameters = validateParameters;
+p5._validateParameters = validateParameters;
```

Comment:
got it. when you have a chance, can you add a comment here so we remember why it's like this